exchange/exchange.go

```diff
@@ -587,6 +590,28 @@ func (e *exchange) makeBid(Bids []*pbsOrtbBid, adapter openrtb_ext.BidderName) (
 	return bids, errList
 }
 
+// If bid got cached inside `(a *auction) doCache(ctx context.Context, cache prebid_cache_client.Client, targData *targetData, bidRequest *openrtb.BidRequest, ttlBuffer int64, defaultTTLs *config.DefaultTTLs, bidCategory map[string]string)`,
+// a UUID should be found inside `a.cacheIds` or `a.vastCacheIds`. This function returns the UUID along with the internal cache URL
+func (e *exchange) getBidCacheInfo(bid *pbsOrtbBid, auc *auction) (openrtb_ext.ExtBidPrebidCacheBids, bool) {
+	var cacheInfo openrtb_ext.ExtBidPrebidCacheBids
+	var cacheUUID string
+	var found bool = false
+
+	if auc != nil {
+		if bid.bidType == openrtb_ext.BidTypeVideo {
+			cacheUUID, found = auc.vastCacheIds[bid.bid]
+		} else {
+			cacheUUID, found = auc.cacheIds[bid.bid]
+		}
+
+		if found {
+			cacheInfo.CacheId = cacheUUID
+			cacheInfo.Url = e.cache.GetPutUrl() + "?uuid=" + cacheUUID //if there's no UUID, should we still define a url?
```

Comment:
Why are you calling this method `GetPutUrl()`? Wouldn't be `GetGetUrl()`, as it is the URL that clients will use to issue the GET request to get the cached content?